WordPress/src/test/java/org/wordpress/android/ui/jetpackoverlay/JetpackFeatureRemovalOverlayUtilTest.kt

```diff
@@ -156,8 +153,8 @@ class JetpackFeatureRemovalOverlayUtilTest : BaseUnitTest() {
         assertTrue(shouldShowOverlay)
     }
 
-    // @Test
-    @Suppress("MaxLineLength")
+    @Ignore("Test originally ignored in commit a6c3a093e6b64d1dfb34954dd6da6b6012613aef without Ignore annotation")
+    @Test
```

Comment:
This is directly related to the change, as to be able to remove the `@Suppress` annotation the test needs to be annotated with `@Test`, so I just added the `@Ignore` annotation here as looking at the history it felt like that was the intent behind commenting the `@Test` annotation in the first place.

But I think I understand what you mean, this could be a separate commit inside this same branch instead of doing everything in one go, good point!

Reply:
> This is directly related to the change, as to be able to remove the @Suppress annotation the test needs to be annotated with @Test, so I just added the @Ignore annotation here as looking at the history it felt like that was the intent behind commenting the @Test annotation in the first place.

Ah, I see what you mean now, you needed the `@Test` annotation to ignore the max line length violation, but you doing so, you needed to properly ignore the test, instead of having it commented out. Then, you're right, this is kind of a grey commit area... 😅 

Thinking about it a bit, ideally, I would probably first do this additional change first and then the main bulk change. 🤔

> But I think I understand what you mean, this could be a separate commit inside this same branch instead of doing everything in one go, good point!

🥇 ❤️